vmss_images/vmss-update.md

```diff
@@ -0,0 +1,72 @@
+# Working with VMSS and Custom Images
+
+
+## Step One - Create two virtual machines and install HTTPD
+
+Run following playbook:
+
+```
+ansible-playbook 01-create-vms.yml --extra-vars "resource_group=myrg"
+```
+
+It will:
+- create 2 virtual machines
+- install HTTPD on both of them
+- change index.html to contain **Image A** and **Image B** respectively
+
+Copy and paste IP addresses of both VMs to the browser:
+
+![Public IP Address](images/vmss-update-vms-ip-addresses.png)
+
+
+You should see two different versions of VM:
+
+![VM A](images/vmss-update-browser-vma.png)
```

Comment:
it's great to have image in doc. could refine the image a little? it's hard to tell something from ip.